Inputs.md

```diff
@@ -0,0 +1,102 @@
+This guide will describe how inputs from the user (keyboard, mouse, etc.) are routed through InputWritr and how the DeviceLayr Gamepad can be used on top of it.
+
+### Table of Contents
+1. [InputWritr](#inputwritr)
+    1. [Events and Triggers](#events-and-triggers)
+    2. [Running Events](#running-events)
+2. [DeviceLayr](#devicelayr)
+
+# InputWritr
+
+## Events and Triggers
+
+InputWritr is a module that automates interactions with user inputs and events. 
+
+Inputs are stored as both character codes (e.g. 37) and as a string representation of themselves (e.g. "left").
+Triggers are what tie inputs with events and are specific actions on the inputs (e.g. press).
+
+Events can be added using `addEvent` which takes in a trigger name, key code, and the function to run when the event is triggered.
+
+```typescript
+InputWritr.addEvent("onKeyDown", 37, () => console.log("left button pressed"););
```

Comment:
Just noticed many of these samples are missing the conventional `e` in name. `InputWriter` vs `InputWritr`.